adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala

```diff
@@ -835,8 +1097,16 @@ class ADAMContext(@transient val sc: SparkContext) extends Serializable with Log
     FragmentRDD(rdd, sd, rgd)
   }
 
-  def loadVcfAnnotations(
-    filePath: String): DatabaseVariantAnnotationRDD = {
+  /**
+   * Loads variant annotations from a VCF.
+   *
```

Comment:
Little minds, consistency, etc.  😉  Reads the same as the other similar methods.